source/style-guide/markup/format/headings.txt

```diff
@@ -0,0 +1,128 @@
+=====================
+Headings and Sections
+=====================
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+|rst| uses repeating characters beneath the heading text to set
+headings.
+
+Heading Level Markup
+--------------------
+
+For, the under- and overlines for each Heading Level, use:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 10 10 60 10
+
+   * - Heading Level
+     - Character
+     - Under or Overline
+     - Common Usage
+     - Render in HTML as
+
+   * - 1
+     - ``=`` (Equals sign)
+     - Both
+     - Page Titles Only
+     - ``<h1>``
+
+   * - 2
+     - ``-`` (Hyphen)
+     - Under
+     - Main Headings
+     - ``<h1>``
+
+   * - 3
+     - ``~`` (Tilde)
+     - Under
+     - Subheadings
+     - ``<h3>``
+
+   * - 4
+     - `` ` `` (Backtick)
+     - Under
+     - 2nd Level Subheadings
+     - ``<h4>``
+
+   * - 5
+     - ``+`` (Plus sign)
+     - Under
+     - 3rd Level Subheadings
+     - ``<h5>``
+
+Tools don't return build errors if you use a different underline order
+as long as it's done consistently. That said, don't use different
+underline orders. It will confuse other writers.
```

Comment:
```suggestion
Tools don't return build errors if you use a different underline order
as long as it's done consistently. However, to avoid confusing other
writers, don't use different underline orders.
```